Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+public class Lottery {
+    public static ball getRandomBall(){

```

Comment:
The method name `getRandomBall` should start with an uppercase letter if it is intended to be a constructor or should follow camelCase if it is a method. Additionally, the return type `ball` should be capitalized as `Ball` if it refers to a class.